src/github.com/getlantern/balancer/balancer_test.go

```diff
@@ -191,13 +193,16 @@ func TestCheck(t *testing.T) {
 	assert.NoError(t, err)
 
 	// recheck failed dialer
+	before := atomic.LoadUint32(&checkCount)
+	// Can't rely on WaitGroup to exactly match the check count. It's time sensitive.
+	// Adds a large enough delta to avoid wg.Done() panicking.
```

Comment:
Oh I thought 5 would be large enough. How about 50? ^^

Let me think a bit about the way to avoid useless Add here